lib/src/webcrypto/webcrypto.ecdh.dart

```diff
@@ -14,43 +14,223 @@
 
 part of 'webcrypto.dart';
 
+/// Elliptic Curve Diffie-Hellman (ECDH) is a key agreement protocol that allows
+/// two parties to establish a shared secret over an insecure channel.
+/// 
+/// An [EcdhPrivateKey] instance represents a private key that can be used to
+/// derive a shared secret with a public key.
+/// 
+/// Instances of [EcdhPrivateKey] can be imported from:
+/// * PKCS8 Key using [EcdhPrivateKey.importPkcs8Key], and,
+/// * JSON Web Key using [EcdhPrivateKey.importJsonWebKey].
+/// 
+/// A random key pair can also be generated using [EcdhPrivateKey.generateKey].
```

Comment:
```suggestion
/// A random key pair can be generated using [EcdhPrivateKey.generateKey].
```
But I might also be careful to say random, I think that's actually what the `AesCbcSecretKey` is. But it's a verb I would be careful of using around a key pair. Since the private key does match the public key, is it random?

I'd suggest something like we have on [RsaPssPrivateKey](https://pub.dev/documentation/webcrypto/latest/webcrypto/RsaPssPrivateKey-class.html) or [RsassaPkcs1V15PrivateKey](https://pub.dev/documentation/webcrypto/latest/webcrypto/RsassaPkcs1V15PrivateKey-class.html).

```suggestion
/// A public-private [KeyPair] consisting of a [EcdhPublicKey] and
/// a [EcdhPrivateKey] can be generated using [EcdhPrivateKey.generateKey].
```

I don't know if the private key in ECDH can be called random. That might actually be possible. I think in RSA, it would be better to not say random, because an RSA key is derived from random input. But the actual key consists of primes, so a key is not random data.

In AES I think we can say random, because the key generation logic is just generating random bytes :rofl: 

Unless, we're sure the same applies here, I would avoid making the claim.